### ISSUE TITLE
The extension doesn't detect an SVG image

### ISSUE THREAD

**Issue by rado84-github** [jscher2000/View-Image-Info-Reborn-for-Firefox#7]
The extension doesn't detect an SVG image which is used as a logo on the page of a local newspaper. I've seen that happen on other pages as well but so far I've been able to find other ways to download the image. But this case puzzled me. Sometimes using uBlock Origin's function to block an image reveals its true URL address, but not in this case. The page is https://telegraph.bg/ and the logo is on top of it. I suppose I could take a screenshot of it and use it for what I need it but I'd still prefer to have it as an SVG image.

**Comment by jscher2000** [comment]
Yes, supporting SVGs was/is a planned feature. I borrowed some code from the behind! extension when I was working on that, but ran into some problems and never circled back to it:

https://github.com/jscher2000/View-Image-Info-Reborn-for-Firefox/blob/main/view-image-info-content-proximate.js#L50

I think behind! may work for your use case of downloading the SVG but haven't tested recently: 

https://addons.mozilla.org/firefox/addon/behind/

**Comment by rado84-github** [comment]
It did work, thanks for the suggestion. :)
